cmd/nri-prometheus/config.go

```diff
@@ -76,6 +76,7 @@ func setViperDefaults(viper *viper.Viper) {
 	viper.SetDefault("auto_decorate", false)
 	viper.SetDefault("insecure_skip_verify", false)
 	viper.SetDefault("standalone", true)
+	viper.SetDefault("enable_autodiscovery", true)
```

Comment:
shouldn't it be "**disable**_autodiscovery"?

Reply:
it is the same, but `disable_autodiscovery` default would be `false` and we would check if we need to add targets with: 
```
if !disable_autodiscovery{
}
```

isn't that less clear? do we usually avoid to use `enable_feature` having true as default?

Reply:
I am changing it, I thought about it and I guess you are right it is more clear for the user with the disable

Reply:
@ardias Updated :)